runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -245,17 +247,34 @@ private StreamingDataflowWorker(
     Consumer<PrintWriter> getDataStatusProvider;
     Supplier<Long> currentActiveCommitBytesProvider;
     ChannelCache channelCache = null;
+    WindmillStreamPool<GetDataStream> getDataStreamPool = null;
```

Comment:
I don't see where this is used, I think you could just move the definition into the different branches.